mitosheet/mitosheet/api/get_dataframe_as_excel.py

```diff
@@ -40,7 +40,16 @@ def get_dataframe_as_excel(params: Dict[str, Any], steps_manager: StepsManagerTy
             # Add formatting to the sheet for pro users
             format = steps_manager.curr_step.df_formats[sheet_index]
             if allow_formatting and format['headers']: 
```

Comment:
It feels like this None check might need to be expanded to check the other things, no? Actually -- see the `.get('key', {})` trick below -- we could use it here and avoid all of the None checking here!

Reply:
Might be good to add a test for header color but not for row color

Reply:
Fixed! In the tests, I did include format objects w/ headers and not rows, as well as format objects with both